templates/zerver/app/home.html

```diff
@@ -133,6 +133,8 @@ <h4>{{ _('Nobody has talked about that yet!') }}</h4>
 
 {% include "zerver/app/message_history.html" %}
 {% include "zerver/app/delete_message.html" %}
+{% include "zerver/app/unsubscribe_stream.html" %}
```

Comment:
You wan't require this, as template should be in `static/js/templates` directory.